frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -294,14 +294,23 @@ function updateToolBarActions(
 }
 
 function getDetailsFields(apiRun?: ApiRun): Array<KeyValue<string>> {
+  // check if the run has finished or not. The default value for apiRun.finished_at is
+  // Date(0), when it is not specified.
+  var finished_at = new Date(0);
+  if (apiRun?.finished_at) {
+    finished_at = new Date(apiRun?.finished_at);
+  }
+  var time_zero = new Date(0);
```

Comment:
The variable name "time zero" is a little unclear/confusing. I would suggest to name as "defaultTime".